### PR TITLE
FMC allow pictures of scramble

### PR DESCRIPTION
**Pull request by Rouxles**

### CODE REVIEW

**Comment by xsrvmy** [comment]
This sounds like only the extra paper may have the scramble image.
Or is it assumed that the solution sheet must be the official one and will have the scramble image?

**Comment by Nevseros** [comment]
> This sounds like only the extra paper may have the scramble image. Or is it assumed that the solution sheet must be the official one and will have the scramble image?

I don't see the problem here -- these sheets are not necessarily two differnt papers.

File: wca-regulations.md [hunk 1]
```diff
@@ -514,7 +514,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - E2e) The competitor's solution must not be directly derived from any part of the scramble sequence. Penalty: disqualification of the attempt (DNF), at the discretion of the WCA Delegate.
         - E2e1) The WCA Delegate may ask the competitor to explain the purpose of each move in their solution, irrespective of the scramble sequence. If the competitor cannot give a valid explanation, the attempt is disqualified (DNF).
 - E3) The competitor may use the following objects during the attempt. Penalty for using unauthorized objects: disqualification of the attempt (DNF).
-    - E3a) Paper (supplied by the judge) and pens/pencils or similar (supplied by the judge, or optionally self-supplied).
+    - E3a) Solution Sheet (supplied by the judge), paper (supplied by the judge) with a picture of the scramble for that attempt, and pens/pencils or similar (supplied by the judge, or optionally self-supplied).
```
Comment (Rouxles) [inline wca-regulations.md]:
```suggestion
    - E3a) Paper (supplied by the judge) with a picture of the scramble for that attempt, the solution sheet (supplied by the judge), and pens/pencils or similar (supplied by the judge, or optionally self-supplied).
```

**Comment by sam596** [comment]
Agreed, this heavily implies that extra paper can only be supplied if it contains a picture of the scramble.

Suggest:
> Paper (supplied by the judge), the solution sheet (supplied by the judge), a picture of the scramble for that attempt (which may be included on the paper or solution sheet), and pens/pencils or similar (supplied by the judge, or optionally self-supplied).

File: wca-regulations.md [hunk 2]
```diff
@@ -514,7 +514,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - E2e) The competitor's solution must not be directly derived from any part of the scramble sequence. Penalty: disqualification of the attempt (DNF), at the discretion of the WCA Delegate.
         - E2e1) The WCA Delegate may ask the competitor to explain the purpose of each move in their solution, irrespective of the scramble sequence. If the competitor cannot give a valid explanation, the attempt is disqualified (DNF).
 - E3) The competitor may use the following objects during the attempt. Penalty for using unauthorized objects: disqualification of the attempt (DNF).
-    - E3a) Paper (supplied by the judge) and pens/pencils or similar (supplied by the judge, or optionally self-supplied).
+    - E3a) Paper (supplied by the judge) with a picture of the scramble for that attempt, the solution sheet (supplied by the judge), and pens/pencils or similar (supplied by the judge, or optionally self-supplied).
```
Comment (Rouxles) [inline wca-regulations.md]: Sam's suggestion:
```suggestion
    - E3a) Paper (supplied by the judge), the solution sheet (supplied by the judge), a picture of the scramble for that attempt (which may be included on the paper or solution sheet), and pens/pencils or similar (supplied by the judge, or optionally self-supplied).
```

Comment (OJCubing) [reply wca-regulations.md]: Very much in favour of this wording 👍

**Comment by Goosly** [comment]
What does this PR attempt to fix?

**Comment by OJCubing** [comment]
Under the current regulations, having an image of the scramble net is not allowed. [E2a](https://wcaregs.netlify.app/#E2a) allows you to distribute scramble *sequences* (not images) and [E3](https://wcaregs.netlify.app/#E3) explicitly lists all authorised objects, of which the scramble net is not. Obviously they should be allowed though.

**Review by Nevseros — APPROVED** [review]

**Pull request merged**